core/src/test/java/org/apache/accumulo/core/clientImpl/TableOperationsHelperTest.java

```diff
@@ -327,23 +327,15 @@ public void testAttachIterator() throws Exception {
             "table.iterator.minc.otherName.opt.key=value",
             "table.iterator.scan.otherName=20,some.classname",});
 
-    try {
-      t.attachIterator("table", setting);
-      fail();
-    } catch (AccumuloException e) {
-      // expected, ignore
-    }
+    IteratorSetting finalSetting = setting; // effectively final var needed for lambda below
```

Comment:
Could probably remove the explanation by just being explicit about the final, when you encounter these. For example:

```java
    final var finalSetting = setting;
```

However, a better idea is to use new variables. I looked at the lines above, and it looks like there's no good reason they need to reuse the variable name. It just as easily could have had IteratorSetting setting1, setting2, etc., or more specific names if there are any that make sense.

Reply:
Addressed in 2c61494